internal/testrunner/runners/system/runner.go

```diff
@@ -275,7 +278,7 @@ func (r *runner) Run(ctx context.Context, options testrunner.TestOptions) ([]tes
 		if err != nil {
 			return result.WithError(fmt.Errorf("failed to prepare scenario: %w", err))
 		}
-		return r.validateTestScenario(ctx, result, scenario, testConfig, serviceOptions)
+		return r.validateTestScenario(ctx, result, scenario, testConfig)
```

Comment:
This parameter can be removed after some refactors in two steps.
These are the commits in two steps:
- https://github.com/elastic/elastic-package/pull/1840/commits/d20c1827635e0619f5431ff3664861bd214535b5
- https://github.com/elastic/elastic-package/pull/1840/commits/bcdaf8a401b2b4c8348bd67771399dbd2f1b2494